guides/deployment/releases.md

```diff
@@ -19,7 +19,7 @@ $ elixir -v
 
 If you are not familiar with Elixir releases yet, we recommend you to read [Elixir's excellent docs](https://hexdocs.pm/mix/Mix.Tasks.Release.html) before continuing.
 
-Once that is done, you can assemble a release by going through all of the steps in our general [deployment guide](/deployment.html) with `mix release` at the end. Let's recap.
+Once that is done, you can assemble a release by going through all of the steps in our general [deployment guide](https://hexdocs.pm/phoenix/deployment.html) with `mix release` at the end. Let's recap.
```

Comment:
Good spot, the issue here is actually the `/` in `/deployment.html`

Could we change it to this instead?

`general [deployment guide](deployment.html) `

This ensures it will work outside of hexdocs too.

Reply:
Done on fd47add I also changed the `Up and Running guide` to follow the same approach.

With the relative path they won't work in the github repo though as you can see it [here](https://github.com/scudelletti/phoenix/blob/ds-fix-release-link/guides/deployment/releases.md#releases-assemble).